test/new-e2e/tests/installer/package_definitions.go

```diff
@@ -0,0 +1,136 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-present Datadog, Inc.
+
+// Package installer contains tests for the datadog installer
+package installer
+
+import (
+	"fmt"
+	"github.com/DataDog/datadog-agent/test/new-e2e/pkg/runner"
+	"github.com/DataDog/datadog-agent/test/new-e2e/pkg/runner/parameters"
+	e2eos "github.com/DataDog/test-infra-definitions/components/os"
+	"os"
+	"strings"
+)
+
+const (
+	// StableVersion the latest stable version of the Datadog Installer
+	StableVersion string = "7.56.0-installer-0.4.5"
```

Comment:
Not a fan if this means we have to create a PR to change this on every release. Can we generate this dynamically?

Reply:
The existing code was fine with not bumping it, is it still the case here? If we don't need to bump it for each release feel free to ignore this thread

We can generate it dynamically with something like `crane ls public.ecr.aws/datadog/installer-package | sort | tail -n 2 | head -n 1`

Reply:
`7.55.0-installer-0.2.1-1` was definitely not the latest 😅